lib/search_click_tracking/service.rb

```diff
@@ -0,0 +1,86 @@
+# frozen_string_literal: true
+
+require 'uri'
+require 'common/client/base'
+require 'common/client/concerns/monitoring'
+require 'common/client/errors'
+require 'search_click_tracking/response'
+require 'search_click_tracking/configuration'
+
+module SearchClickTracking
+  # This class builds a wrapper around Search.gov web results API. Creating a new instance of class
+  # will and calling #results will return a ResultsResponse upon success or an exception upon failure.
+  #
+  # @see https://search.usa.gov/sites/7378/api_instructions
+  #
+  class Service < Common::Client::Base
+    include Common::Client::Concerns::Monitoring
+
+    STATSD_KEY_PREFIX = 'api.search_click_tracking'
+
+    configuration SearchClickTracking::Configuration
+
+    attr_reader :url
+    attr_reader :query
+    attr_reader :position
+    attr_reader :client_ip
+    attr_reader :user_agent
+
+    def initialize(url, query, position, client_ip, user_agent)
+      @url = url
+      @query = query
+      @position = position
+      @client_ip = client_ip
+      @user_agent = user_agent
+    end
+
+    # POSTs click tracking query param data to search.gov
+    #
+    def track_click
+      Faraday.post(url_with_params, '')
```

Comment:
Can you use the post method defined in the common/client/base class?

Reply:
This is a weird 3rd party request that returns an empty body or a string. Using the perform / post method defined in the base class I was getting some unwanted errors as it tried to parse the body. @dillo helped me come up with this solution and may be able to explain it better.

Reply:
Ok strange, so passing in the query params [here](https://github.com/department-of-veterans-affairs/vets-api/blob/master/lib/common/client/base.rb#L94) as the third parameter to the method wasn't returning the expected results?

Reply:
Correct. Passing in the params here allowed us to send the request properly, but somewhere in the way that was handling the response was causing it to fail when we got an empty response back. I can explore this further.